apps/arena/priv/config.json

```diff
@@ -70,6 +70,772 @@
         "radius": 365.55,
         "shape": "circle",
         "vertices": []
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "radius": 0.0,
+        "shape": "polygon",
+        "vertices": [
+          {
+            "x": -956.0,
+            "y": -779.0
+          },
+          {
+            "x": -761.0,
+            "y": -727.0
+          },
+          {
+            "x": 219.0,
+            "y": -1232.0
+          },
+          {
+            "x": 105.0,
+            "y": -1632.0
+          },
+          {
+            "x": -503.0,
+            "y": -1416.0
+          },
+          {
+            "x": -995.0,
+            "y": -989.0
+          }
+        ]
+      },
+      {
+        "position": {
+          "x": 0.0,
+          "y": 0.0
+        },
+        "radius": 0.0,
+        "shape": "polygon",
+        "vertices": [
+          {
+            "x": -499.0,
+            "y": 1844.0
+          },
+          {
+            "x": 222.0,
+            "y": 1937.0
+          },
+          {
+            "x": 737.0,
+            "y": 1770.0
+          },
+          {
+            "x": 831.0,
+            "y": 1570.0
+          },
+          {
+            "x": 643.0,
+            "y": 1485.0
+          },
+          {
+            "x": -425.0,
+            "y": 1416.0
+          }
+        ]
+      },
+      {
+        "collider": "carapace 1",
```

Comment:
What is this field?

Reply:
this a field that we'll use to identify the rocks in the map, it's really hard to identify where a polygon belongs without this field

Reply:
Can't it be the "name" instead of "collider"? I think that's a word of unity business logic, not ours

Reply:
sure! yeah the name isn't important, we just want a way to recognize which polygon correspond with which collider